File: PopAppCore/Entities/ScheduleContainerFreigth.cs
using PopAppCore.Enums;
using System.ComponentModel.DataAnnotations;

namespace PopAppCore.Entities
{
    public class ScheduleContainerFreigth
    {
        [Key]
        public int Id { get; set; }
        public int ScheduleContainerId { get; set; }
        public int FreigthId { get; set; }
        public virtual ScheduleContainer ScheduleContainer { get; set; }
        public virtual Freigth Freigth { get; set; }
        public EvaluationStatus EvaluationStatus { get; set; }
        public string EvaluationMessage { get; set; }
    }
}

File: PopApp/ClientApp/src/app/models/schedule/schedule-container-freigth.ts
import { FreigthEvaluationStatus } from '../../enums/freigth-evaluation-status';
import { Freigth } from '../freigth/freigth';
import { ScheduleContainer } from './schedule-container';

export interface ScheduleContainerFreigth {
  id?: number;
  scheduleContainerId?: number;
  freigthId?: number;
  scheduleContainer?: ScheduleContainer;
  freigth?: Freigth;
  evaluationStatus?: FreigthEvaluationStatus;
  evaluationMessage?: string;
}
